app/controllers/file_uploads_controller.rb

```diff
@@ -2,12 +2,7 @@ class FileUploadsController < ApplicationController
 
   # The second value for each category entry will be used to determine the job class that processes the data.
   # Ex: Selecting "Spawning Success" in the form will post "SpawningSuccess" and process the data with a SpawningSuccessJob.
-  CATEGORIES = [
-      ['Spawning Success','SpawningSuccess'],
-      ['Tagged Animal Assessment','TaggedAnimalAssessment'],
-      ['Untagged Animal Assessment', 'UntaggedAnimalAssessment'],
-      # ['Wild Collection','WildCollection']
-  ].freeze
+  CATEGORIES = CsvImporter::CATEGORIES.map {|category| [category, category.delete(' ')] }.freeze
```

Comment:
I had a little trouble realizing that `CATEGORIES` is a completely different structure from `CsvImporter::CATEGORIES`.
Do you think it would disambiguate if we call this something like
```ruby
FILE_UPLOAD_CATEGORIES = CsvImporter::CATEGORIES.map {|category| [category, category.delete(' ')] }.freeze
```
?

Reply:
Absolutely! Good point.

Reply:
Absolutely! Good point.